documentation/asciidoc/accessories/camera/libcamera_apps_getting_started.adoc

```diff
@@ -55,10 +55,6 @@ If you do need to add your own `dtoverlay`, the following are currently recognis
 | `dtoverlay=ov9281`
 |===
 
-To override the automatic camera detection, _Bullseye_ users will also need to delete the entry
-
-`camera_auto_detect=1`
-
-if present in the `config.txt` file. If you need to do edit this file then your Raspberry Pi will need to be rebooted. 
+To override the automatic camera detection, _Bullseye_ users will also need to delete the entry `camera_auto_detect=1` if present in the `config.txt` file. If you need to do edit this file then your Raspberry Pi will need to be rebooted. 
```

Comment:
"to do edit" -> "to edit" ? And perhaps something like "for this change to take effect" is needed at the end?
Or maybe just "Your Raspberry Pi will need to be rebooted after editing this file." ? :man_shrugging:

Reply:
@lurch 
If we need correction, https://github.com/raspberrypi/documentation/pull/2450

Reply:
@lurch 
Thanks